lib/compiler/filters/static-filter-checker.js

```diff
@@ -68,46 +76,55 @@ var checks = {
 var ALLOWED_OP_FORMS = {
     '==': [
         { left: checks.isField, right: checks.isSimpleLiteral },
-        { left: checks.isSimpleLiteral, right: checks.isField }
+        { left: checks.isSimpleLiteral, right: checks.isField },
+        { left: checks.isPropertyAccess, right: checks.isSimpleLiteral }
```

Comment:
Why not allow `isPropertyAccess` everywhere `isField` is allowed? That sounds like a logical thing to do to me.

If we go that route we should probably fold the `isPropertyAccess` check into `isField` to avoid duplication. The `isField` check would then mean “field, either plain or nested”.